### DISCUSSION TITLE
I Have raised a ticket last 10 days ago because my account got disabled i did 'get replay from GitHub Team.

### DISCUSSION THREAD

**Discussion by 0xmanoj** [community/community#110204] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

My Account got Disabled without reason i have raised a ticket to get beck it is more imported to me my company project is there please check and and give back my account🥲🥲

username : manugowda18
Ticket Id : #2604757

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @0xmyth !  The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them. You are more likely to get a useful response if you are posting your question in the applicable category. 

However, the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
